cli/src/semgrep/commands/wrapper.py

```diff
@@ -4,12 +4,26 @@
 from typing import Callable
 from typing import NoReturn
 
+import click
+
 from semgrep.error import FATAL_EXIT_CODE
 from semgrep.error import SemgrepError
 from semgrep.state import get_state
 from semgrep.verbose_logging import getLogger
 
 
+class CommandSection(click.Command):
+    section: str = (
+        "Commands"  # Name of the section to group the command under in the help menu
+    )
+    priority: int = 0  # Order of the command group (lowest is first)
+
+
+class AdvancedCommand(CommandSection):
```

Comment:
again, not useful as all of that is now handled by osemgrep.
If you want to improve the CLI UX, you need to do it on the OCaml side first.

Reply:
This code change is only used for generating the help message (ordering the sections).